### PR TITLE
Variant export

### PR DESCRIPTION
**Pull request by WilliamRichards2017**
Fixing issues with exporting and importing variants.  Single click to save and download export file, fixed issue with variants getting bypassed when exporting as vcf, fixed issue where only last flagged variant got saved to vcf, fixed issue where genes weren't loaded on variant import, added description to what is getting exported in export dialog

### CODE REVIEW

**Comment by anderspitman** [comment]
quarantest build successful. Demo link at
http://efc1dc63.quarantest.iobio.io

**Comment by WilliamRichards2017** [comment]
To test:
1. Run with demo data - let the data load
2. Export variants - it should only be a single click on save to download the export file.  Save both as csv and vcf. You should receive no error messages about bypassing a gene.
3. Open the downloaded vcf, the vcf should contain 4 non-header lines, 1 for each variant in the left panel
4. Reload gene and select load data.  Load using the WES demo data
5. Select import variants, and import the gene.iobio csv generated from step 2
6. All variants should load, and all three genes should be loaded in the left panel gene tab

File: client/app/components/partials/ExportVariants.vue [hunk 1]
```diff
@@ -4,36 +4,36 @@
...
-      margin-bottom: 0px
+      margin-bottom: 0
 
   .radio-group.radio-group--column
-    margin-top: 0px
-    padding-top: 0px
-    padding-bottom: 0px
+    margin-top: 0
+    padding-top: 0
+    padding-bottom: 0
```
Comment (adityaekawade) [inline client/app/components/partials/ExportVariants.vue:19]: This is not related to the issue or is causing any side effects but I was just wondering if these changes are caused by IDE/ linting?

**Review by adityaekawade — APPROVED** [review]
Tried on quarantest with the steps mentioned. Works as expected. Reviewed variant and added note showed up after importing variants from the CSV file.

File: client/app/components/partials/ExportVariants.vue (hunk 1 above)
Comment (WilliamRichards2017) [reply client/app/components/partials/ExportVariants.vue:19]: These were done by hand, I've been going through and trying to clean up some of the ide warning messages.  I've found a few tricky issues that were picked up by useful warnings, but they get buried in some of the trivial ones, so I've been trying to boil down the warnings to the ones that are potentially important

**Comment by anderspitman** [comment]
quarantest build successful. Demo link at
http://322ed115.quarantest.iobio.io
